### DISCUSSION TITLE
Error loading images in a new project at https://www.makesense.ai/

### DISCUSSION THREAD

**Discussion by armbrustsamuel** [SkalskiP/make-sense#225] [General]
Hi colleagues, I am facing an issue since two days ago. While I try to load any number of images in a new project, the images seem to be loading but it never appears.

![image](https://user-images.githubusercontent.com/8709701/149800896-ade5df2c-ffaa-4d0f-ba54-e8b48150c4dd.png)

I saw in the console the some error, such as:
blob:https://www.makesense.ai/341d07df-a75d-4516-bc31-c02898646847:1 GET blob:https://www.makesense.ai/341d07df-a75d-4516-bc31-c02898646847 net::ERR_ACCESS_DENIED 200 (OK)

Do you have any idea how to solve it?

**Comment by SkalskiP** [comment]
Hi, @armbrustsamuel 👋 ! I'm really sorry for the late response. Are you still experiencing this issue?

**Reply by rasyidf** [reply]
are you using edge? did you activate strict security.?

**Comment by zupdaz** [comment]
I've got the same problem @SkalskiP. Most of pictures never load, some do however. Is this repo still maintained?

**Reply by rasyidf** [reply]
It's getting harder to try helping maintaining this repo after last few versions, the git becoming too big to clone.
